crates/store/re_types/definitions/rerun/blueprint/archetypes/dataframe_query.fbs

```diff
@@ -49,3 +50,38 @@ table DataframeQuery (
     time_range_queries: rerun.blueprint.components.TimeRangeQueries ("attr.rerun.component_optional", nullable,order: 500);
 }
 
+
+
+/// The query for the dataframe view.
+table DataframeQueryV2 (
+    "attr.rerun.scope": "blueprint"
+) {
+    // --- Optional ---
+
+    /// The timeline for this query.
+    ///
+    /// If unset, the time panel's timeline is used and stored.
+    timeline: rerun.blueprint.components.TimelineName ("attr.rerun.component_optional", nullable, order: 100);
+
+    /// If set, a range filter is applied.
+    ///
+    /// Note: will be unset as soon as `timeline` is changed.
+    //TODO(ab): should this use the list-with-timelines trick?
+    range_filter: rerun.blueprint.components.RangeFilter ("attr.rerun.component_optional", nullable, order: 200);
+
+    /// Whether the filter by event feature is active.
```

Comment:
should probably expand here what the feature does

Reply:
Mmm, this indirectly points at a shortcoming of our doc codgen:

<img width="590" alt="image" src="https://github.com/user-attachments/assets/5f681416-f69c-40a6-ac7c-69fde1816acc">

The original fields of the query are shown, but that is _not_ what the (custom) constructor accepts (in this case, a single `filter_by_event` argument which, when provided, sets active to `True` and column to the proper (parsed) value.

Fixing codegen for that seems... ambitious (need to parse the custom init args, etc.). Looks like I need to roll a datatype that contains the bool and the column.

(Note that I really want a bool + column, and just not an optional column. That way, the user can toggle the check box to see the effect without its filter column being "forgotten" each time.)

Reply:
Fixed.